### PR TITLE
Export /opt/intel

### PR DESCRIPTION
**Pull request by sean-smith**
Signed-off-by: Sean Smith <seaam@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: recipes/intel_install.rb [hunk 1]
```diff
@@ -71,24 +71,6 @@
   modulefile "2019.4"
 end
 
-# intel performance primitives
-create_modulefile "#{node['cfncluster']['modulefile_dir']}/intelipp" do
```
Comment (demartinofra) [inline recipes/intel_install.rb]: was this supposed to be part of this PR/commit?

File: recipes/_compute_base_config.rb [hunk 2]
```diff
@@ -47,6 +47,14 @@
   action %i[mount enable]
 end
 
+# Mount /opt/intel over NFS
+mount '/opt/intel' do
+  device "#{nfs_master}:/opt/intel"
+  fstype 'nfs'
+  options 'hard,intr,noatime,vers=4,_netdev'
```
Comment (demartinofra) [inline recipes/_compute_base_config.rb]: so far we've been using nfs version 3 everywhere. I agree we could update if there are no drawbacks but I would rather do it for all the mount points (I recall there were some performance degradations with version 4 we want to double check)

File: recipes/_master_base_config.rb [hunk 3]
```diff
@@ -112,6 +112,13 @@
   options ['no_root_squash']
 end
 
+# Export /opt/intel
+nfs_export "/opt/intel" do
```
Comment (demartinofra) [inline recipes/_master_base_config.rb]: are we exporting and mounting this on all OSes?

File: recipes/intel_install.rb (hunk 1 above)
Comment (sean-smith) [reply recipes/intel_install.rb]: yes, this removes un-needed module files. These components are all loaded in the psxe module file.

File: recipes/_master_base_config.rb (hunk 3 above)
Comment (sean-smith) [reply recipes/_master_base_config.rb]: Changed to only exporting if the `/opt/intel` directory exists, this is the case on

* alinux, ubuntu1604, centos7
* not in china

File: recipes/_compute_base_config.rb (hunk 2 above)
Comment (sean-smith) [reply recipes/_compute_base_config.rb]: Can you send me a link to that performance issue? I don't have any context for that.

**Pull request merged**